### DISCUSSION TITLE
bug in github login page "https://github.com/signup"

### DISCUSSION THREAD

**Discussion by manjuu-eater** [community/community#42582] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I found a bug on GitHub new account creating page.

+ start web browser
+ open page "https://github.com/signup"
+ create new tab in same browser, and open page "https://github.com/login"
+ click "Sign in" button (input can be empty)
+ back to the tab of "https://github.com/signup"
+ enter some text in email form
+ email address check fails, and error page "What‽" appears

Sorry if reporting place is wrong.

**Comment by radar-blip** [comment, marked as answer]
You can file a bug report here: [Bug-Report](https://support.github.com/contact/bug-report)

**Reply by manjuu-eater** [reply]
Thank you very much!!!!
I couldn't find out this page...
Very difficult to find.

I'm going to report the bug to the form.

**Reply by radar-blip** [reply]
No problem. 
You didn't exactly post in the wrong place but the bug-form should get better results.
